rclpy/rclpy/executors.py

```diff
@@ -720,7 +722,11 @@ def spin_once(self, timeout_sec: float = None) -> None:
         else:
             handler()
             if handler.exception() is not None:
-                raise handler.exception()
+                rclpy.logging._root_logger.log(
+                    'Exception: ' + str(handler.exception()),
+                    LoggingSeverity.WARN,
+                    name='SingleThreadedExecutor',
+                )
```

Comment:
I think we should provide the same experience for `SingleThreadedExecutor`.

Reply:
makes sense to me